Paralayout/GeometryAdditions.swift

```diff
@@ -181,7 +181,37 @@ extension CGRect {
 // MARK: -
 
 extension UIEdgeInsets {
-    
+
+    /// Initialize edge insets with the same amount inset on each edge.
+    ///
+    /// - parameter inset: The amount by which to inset each edge.
+    public init(uniform inset: CGFloat) {
+        self.init(top: inset, left: inset, bottom: inset, right: inset)
+    }
+
+    /// Initialize edge insets with the same amount outset on each edge.
+    ///
+    /// This is the inverse of `init(uniform:)`.
+    ///
+    /// - parameter outset: The amount by which to outset each edge.
+    public init(uniformOutset outset: CGFloat) {
```

Comment:
This one always felt silly/repetitive to me, but don't feel strongly.

Reply:
I sort of agree, but it's also one that we use in enough places internally that it felt like it proved its usefulness.